selfdrive/athena/registration.py

```diff
@@ -28,7 +28,7 @@ def register(show_spinner=False) -> str:
 
   IMEI = params.get("IMEI", encoding='utf8')
   HardwareSerial = params.get("HardwareSerial", encoding='utf8')
-  dongle_id = params.get("DongleId", encoding='utf8')
+  dongle_id: str = params.get("DongleId", encoding='utf8')
```

Comment:
In #24379 I made `dongle_id`'s type and `register`'s return type `Optional[str]` - that function makes no guarantee to return `str`

https://github.com/commaai/openpilot/blob/06976a7f94d49ea2b82d7163a9603284cac80be4/selfdrive/athena/registration.py#L32

https://github.com/commaai/openpilot/blob/06976a7f94d49ea2b82d7163a9603284cac80be4/selfdrive/athena/registration.py#L26